brozzler/worker.py

```diff
@@ -549,7 +567,8 @@ def brozzle_site(self, browser, site):
                         page,
                     )
                     self._frontier.completed_page(site, page)
-                    page = None
```

Comment:
do we really want to skip this line?

Reply:
Good catch. You're correct that this seems necessary in disclaim_site() when we're done retrying a page.